flink-tests/src/test/java/org/apache/flink/test/state/operator/restore/AbstractOperatorRestoreTestBase.java

```diff
@@ -79,16 +62,20 @@
  */
 public abstract class AbstractOperatorRestoreTestBase extends TestLogger {
 
+	private static final int NUM_TMS = 2;
```

Comment:
the existing test only uses 1 task manager.

Reply:
fixing